crates/core/src/client/helpers.rs

```diff
@@ -621,210 +595,73 @@ mod tests {
-            )
-            .unwrap(),
-            FieldElement::from_hex_be(
-                "0000000000000000000000000000000000000000000000000000000000000000",
-            )
-            .unwrap(),
-            FieldElement::from_hex_be(
-                "0000000000000000000000000000000000000000000000000000000000000000",
-            )
-            .unwrap(),
-            FieldElement::from_hex_be(
-                "0000000000000000000000000000000000000000000000000000000000000000",
-            )
-            .unwrap(),
-            FieldElement::from_hex_be(
-                "0000000000000000000000000000000000000000000000000000000000000000",
-            )
-            .unwrap(),
-            FieldElement::from_hex_be(
-                "0000000000000000000000000000000000000000000000000000000000000000",
-            )
-            .unwrap(),
-            FieldElement::from_hex_be(
-                "0000000000000000000000000000000000000000000000000000000000000000",
-            )
-            .unwrap(),
-            FieldElement::from_hex_be(
-                "0000000000000000000000000000000000000000000000000000000000000000",
-            )
-            .unwrap(),
-            FieldElement::from_hex_be(
-                "0000000000000000000000000000000000000000000000000000000000000000",
-            )
-            .unwrap(),
-            FieldElement::from_hex_be(
-                "0000000000000000000000000000000000000000000000000000000000000000",
-            )
-            .unwrap(),
-            FieldElement::from_hex_be(
-                "0000000000000000000000000000000000000000000000000000000000000000",
-            )
-            .unwrap(),
-            FieldElement::from_hex_be(
-                "0000000000000000000000000000000000000000000000000000000000000000",
-            )
-            .unwrap(),
-            FieldElement::from_hex_be(
-                "0000000000000000000000000000000000000000000000000000000000000000",
-            )
-            .unwrap(),
-            FieldElement::from_hex_be(
-                "0000000000000000000000000000000000000000000000000000000000000000",
-            )
-            .unwrap(),
-            FieldElement::from_hex_be(
-                "0000000000000000000000000000000000000000000000000000000000000000",
-            )
-            .unwrap(),
-            FieldElement::from_hex_be(
-                "0000000000000000000000000000000000000000000000000000000000000000",
-            )
-            .unwrap(),
-            FieldElement::from_hex_be(
-                "0000000000000000000000000000000000000000000000000000000000000002",
-            )
-            .unwrap(),
-            FieldElement::from_hex_be(
-                "00000000000000000000000000000000000000000000000000000000000fffff",
-            )
-            .unwrap(),
+            FieldElement::from_hex_be("0000000000000000000000000000000000000000000000000000000000000002").unwrap(),
```

Comment:
can we add a comment of where this data is sourced from?

can be a separate issue. relates to the question i asked in telegram around the mock data i have for starknet event keys.

Reply:
yes agreed

Reply:
we should ban hardcoded values that arent comprehensive, it triggers me (i wrote this i think) smh